src/Http/Requests/Account/UpdateUserDataRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Account;
+
+use Blumilk\Meetup\Core\Http\Requests\Speaker\Rules\AvatarFileRules;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
+
+class UpdateUserDataRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "email" => ["required", "string", "email", "max:255",
+                Rule::unique("users")->where(fn ($query) => $query->where("email", "!=", Auth::user()->email)), ],
```

Comment:
```suggestion
    {
        $userId = $this->user()->id;
        return [
            "email" => ["required", "string", "email", "max:255", "unique:users,email,{$userId},id"],
```

Reply:
It is shorter type of that rule but u can decide which choose.

Reply:
i was looking for something like this but there was this version in the documentation